frontend/app/routes/_gcweb-app.personal-information.address.edit.tsx

```diff
@@ -1,40 +1,65 @@
-import { Form, Link, useLoaderData } from '@remix-run/react';
+import { type ActionFunctionArgs, type LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { Form, Link, useActionData, useLoaderData } from '@remix-run/react';
 
-import { type LoaderFunctionArgs, type ActionFunctionArgs, json, redirect } from '@remix-run/node';
+import { z } from 'zod';
 
-import { getUserService } from '~/services/user-service.server';
+import { InputField } from '~/components/input-field';
 import { getSessionService } from '~/services/session-service.server';
+import { getUserService } from '~/services/user-service.server';
 import { getEnv } from '~/utils/env.server';
 
 export async function loader({ request }: LoaderFunctionArgs) {
   const env = getEnv();
   const userService = getUserService({ env });
   const userId = await userService.getUserId();
-  return json({
-    user: await userService.getUserInfo(userId),
-  });
+  const userInfo = await userService.getUserInfo(userId);
+
+  if (!userInfo) {
+    throw new Response(null, { status: 404, statusText: 'User Info Not Found' });
```

Comment:
```suggestion
    throw new Response('User info not found', { status: 404 });
```

Reason: the browser has a standard/conventional status text for 404s, no need to override it.

Reply:
Express will send a `Not Found` status by default.. no need to unnecessarily complicate the code.

`throw new Response(null, { status: 404 })` 

becomes:
 
```
HTTP/1.1 404 Not Found
x-remix-catch: yes
Date: Sat, 20 Jan 2024 18:24:57 GMT
Connection: keep-alive
Keep-Alive: timeout=5
Transfer-Encoding: chunked
```